_schema.graphql

```diff
@@ -4596,6 +4615,14 @@ type HomePageModuleContextFair {
     # A tz database time zone, otherwise falls back to `X-TIMEZONE` header
     timezone: String
   ): String
+  active_start_at(
+    # This arg is deprecated, use timezone instead
+    convert_to_utc: Boolean
```

Comment:
<!--
  1 failure:  Addition uses `sn...
  0 warning: 
  
  
  DangerID: danger-id-default;
  File: _schema.graphql;
  Line: 4620;
-->
- :no_entry_sign: Addition uses `snake_case` instead of `camelCase`: `convert_to_utc`